src/engraving/libmscore/mcursor.cpp

```diff
@@ -32,11 +32,13 @@
 #include "libmscore/durationtype.h"
 #include "libmscore/measure.h"
 #include "libmscore/segment.h"
-#include "libmscore/score.h"
+#include "libmscore/masterscore.h"
 #include "libmscore/instrtemplate.h"
 #include "libmscore/keysig.h"
 #include "libmscore/timesig.h"
 
+#include "masterscore.h"
```

Comment:
This one seems unnecessary too, I guess it's the same problem :)

Reply:
Needed, because

Reply:
But there is already `#include "libmscore/masterscore.h"` at line 35.

Reply:
Now, I got it, fixed )